plugin/src/main/java/org/autorefactor/refactoring/Refactorings.java

```diff
@@ -413,4 +418,40 @@ public BadLocationException call() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * Gets the ASTRewrite rewrite object.
+     *
+     * @return the ASTRewrite rewrite
+     */
+    public ASTRewrite getRewrite() {
+        return rewrite;
+    }
+
+    /**
+     * Insert a statement in the end of a method body.
+     *
+     * @param stmt the statement to be added
+     * @param md the method declaration
+     */
+    public void insertLast(Statement stmt, MethodDeclaration md) {
+        insertAt(stmt,
+                md.getBody().statements().size(),
+                Block.STATEMENTS_PROPERTY,
+                md.getBody());
+    }
+
+    /**
+     * Insert a declaration in the end of a type declaration body.
+     *
+     * @param declaration the declaration to be added
+     * @param td the type declaration
+     */
+    public void insertLast(BodyDeclaration declaration, TypeDeclaration td) {
+        insertAt(
+                declaration,
+                td.bodyDeclarations().size(),
+                td.getBodyDeclarationsProperty(),
+                td);
+    }

```

Comment:
Please make use of the new `Refactorings.insertLast()` and either:
1. remove these specific methods
2. or move them back to `AndroidWakeLockRefactoring`.